<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pico5000MRI</class>
 <widget class="QMainWindow" name="pico5000MRI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="PlotWidget" name="picoPlot"/>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="7">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>pre-Trigger points</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="10" colspan="2">
             <widget class="QLineEdit" name="leAcquisitionTime">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Totla aquisition time include pre and post trigger intervals</string>
              </property>
             </widget>
            </item>
            <item row="0" column="9">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>post-Trigger points</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="10">
             <widget class="QSpinBox" name="sbPostTriggerPoints">
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pbPauseContinue">
              <property name="toolTip">
               <string>toggles between pause (red) and capture (green) when pulse sequence is going</string>
              </property>
              <property name="text">
               <string>pause/ continue</string>
              </property>
             </widget>
            </item>
            <item row="0" column="8">
             <widget class="QSpinBox" name="sbPreTriggerPoints">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
             </widget>
            </item>
            <item row="1" column="9">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Aquisiton time(ms)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QComboBox" name="comboBox_2"/>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pbPicoCapture">
              <property name="text">
               <string>Capture</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QDoubleSpinBox" name="dspboxSampleTime">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.001000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label">
              <property name="toolTip">
               <string>digitization time, Time between successive measuremetns</string>
              </property>
              <property name="text">
               <string>Sample time(us)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="11">
             <widget class="QPushButton" name="pbSetupPS">
              <property name="text">
               <string>Setup PS</string>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QComboBox" name="comboBox_3"/>
            </item>
            <item row="1" column="7">
             <widget class="QComboBox" name="comboBox_4"/>
            </item>
            <item row="1" column="8">
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>ChD</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="6">
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string>ChC</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>ChB</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>ChA</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="comboBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="leMessages"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetup"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
